qa/tasks/vstart_runner.py

```diff
@@ -419,7 +419,17 @@ def _asok_path(self):
         # the PID of the launching process, not the long running ceph-fuse process.  Therefore
         # we need to give an exact path here as the logic for checking /proc/ for which
         # asok is alive does not work.
-        path = "./out/client.{0}.{1}.asok".format(self.client_id, self.fuse_daemon.subproc.pid)
+
+        # Load the asok path from ceph.conf as vstart.sh now puts admin sockets
+        # in a tmpdir. All of the paths are the same, so no need to select
+        # based off of the service type.
+        with open("./ceph.conf") as f:
```

Comment:
There's a `config_path` method that gives the ceph.conf path

Reply:
I fixed that and other places where it was hard-coded.